kamon-spray/src/main/scala/kamon/spray/instrumentation/ClientRequestInstrumentation.scala

```diff
@@ -123,9 +123,7 @@ class ClientRequestInstrumentation {
 
         request.asInstanceOf[SegmentAware].segment = segment
 
-        val responseFuture = originalSendReceive.apply(request)
-        responseFuture.map(result ⇒ segment.finish())(SameThreadExecutionContext)
-        responseFuture
+        originalSendReceive.apply(request).andThen { case _ ⇒ segment.finish() }(SameThreadExecutionContext)
```

Comment:
what about :
```scala
val responseFuture = originalSendReceive.apply(request)
responseFuture.onComplete(_ ⇒ segment.finish())(SameThreadExecutionContext)
responseFuture
```
instead of:
``` scala
originalSendReceive.apply(request).andThen { case _ ⇒ segment.finish() }(SameThreadExecutionContext)
```
only for maintain the code consistent in the project.

what you think?